tests/casp/nfs_client.pm

```diff
@@ -0,0 +1,160 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Basic check for NFS client in CASP
+# Maintainer: Tomas Hehejik <thehejik@suse.com>
+# Description: The test is using NFS_SHARE variable for server with value nfs://server.somewhere/some/path
+# and performs some basic NFSv3/v4 passive client tests
+
+use base "opensusebasetest";
+use strict;
+use testapi;
+
+sub casp_etc_strip {
+
```

Comment:
hmm, the first blank line at line 18 after "#include" is imo correct, second one removed.